api/config/settings.py

```diff
@@ -476,3 +478,17 @@
 }
 
 django_heroku.settings(locals())
+
+# Cloudinary settings for Django. Add to your settings file.
+CLOUDINARY = {
+  'cloud_name': 'ireporter',  
+  'api_key': '774853988846169',  
+  'api_secret': 'C4e4QuEgjubsi-utsCc_yhlgnDQ',  
+}
+
+# Cloudinary settings. Run before pycloudinary is used.
+cloudinary.config(
```

Comment:
I think it's better if this data is kept in the .env file

Reply:
Okay, thanks for the note, will make the change.